app/assets/javascripts/nfg_ui/file_inputs.js

```diff
@@ -0,0 +1,5 @@
+// This populates the label inside/below custom file inputs with
+// the file name that was selected.
+$(document).on('change', '.custom-file-input', function(event) {
```

Comment:
@ajporterfield: Should we embrace this within the `if NfgUi.turbolinks`... code?

Reply:
@ajporterfield also, Evo has a solution for this: app/assets/javascripts/bootstrap4/custom_file_input.coffee -- most notably this version accounts for ajax / xhr and modals. Does this version do the same? I am not sure if document on change accounts for xhr / ajax and modals.

Reply:
Since this is a change event (vs. something that needs to be run on page load), we don't need to worry about turbolinks vs. non-turbolinks.

I believe $(document).on will handle the scenarios you brought up (ajax, xhr, modals).  It's definitely something worth verifying though.

My preference would be to remove the Evo js and rely on this from NFG UI since it's something related to BS4 that we'll need in both DMS and Evo.  An alternative would be to just move what we have in Evo to NFG UI and use that instead of this new file.